cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -76,6 +76,9 @@ function vendor_cookbook
 
 # deploy config files
 export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/aws/bin
+# Load ParallelCluster environment variables
+. /etc/profile.d/pcluster.sh
```

Comment:
I think this should be moved after the check on ".bootstrapped" file, because if the AMI is not a PCluster one, the file `/etc/profile.d/pcluster.sh` doesn't exist and the source fails before the check

Reply:
Added a check to verify that the file exists